bin/load-module-esm-support.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var dirnamePath = path.dirname;
+var extnamePath = path.extname;
+var resolvePath = path.resolve;
+
+var cwd = process.cwd();
+var isModulePackage = getPackageType() === 'module';
+
+function getPackageType() {
+    var dir = __dirname;
+
+    while (dir !== '/' && dir !== process.env.HOME) {
+        var pkgPath = resolvePath(dir, 'package.json');
+
+        try {
+            if (fs.existsSync(pkgPath)) {
+                var pkg = require(pkgPath);
+
+                if (pkg.type) {
+                    return pkg.type;
+                }
+            }
+        } catch (e) {}
+
+        dir = dirnamePath(dir);
+    }
+
+    return 'commonjs';
+}
+
+module.exports = function (file) {
+    var ext = extnamePath(file);
+
+    if (ext === '.mjs' || isModulePackage && ext === '.js') {
+        return import(resolvePath(cwd, file));
```

Comment:
```suggestion
        return import(pathToFileURL(resolvePath(cwd, file)));
```

IIRC Windows has issues if you pass a filesystem path to `import`, might be the `:` in absolute paths that causes problems.

Reply:
ugh, this is why we cant have nice things